guide/interactions/context-menus.md

```diff
@@ -0,0 +1,43 @@
+# Context Menus
+
+Context Menus are application commands which appear on right clicking or tapping a user or a message.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read that section first, so that you can understand the methods used in this section.
+:::
+
+## Registering context menu commands
+
+To create a context menu you construct a new `ContextMenuCommandBuilder`. You can then set the type of the context menu (user or message) using the `setType()` method.
+
+```js
+const { ContextMenuCommandBuilder } = require('@discordjs/builders');
+
+const data = new ContextMenuCommandBuilder()
+	.setName('echo')
+	.setType('USER');
+```
+
+## Receiving context menu command interactions
+
+Context Menus are received via an interaction. You can check if a given interaction is a context menu by invoking the `isContextMenu()` method, you can use the `isMessageContextMenu()` and `isUserContextMenu()` methods to check for the specific type of context menu interaction:
```

Comment:
```suggestion
Context Menus are received via `interactionCreate` event. You can check if a given interaction is a context menu by invoking the `isContextMenu()` method, you can use the `isMessageContextMenu()` and `isUserContextMenu()` methods to check for the specific type of context menu interaction:
```

Reply:
this was directly taken from https://discordjs.guide/interactions/modals.html#receiving-modal-submissions also if this change is necessary shouldn't it be `the interactionCreate event`